protocols/route-processor/.env.example

```
@@ -7,4 +7,4 @@ MNEMONIC="test test test test test test test test test test test junk"
 REPORT_GAS=false
 TENDERLY_PROJECT=
 TENDERLY_USERNAME=
-PRIVATE_KEY=
\ No newline at end of file
+PRIVATE_KEY=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The PRIVATE_KEY key should go before the REPORT_GAS key

(UnorderedKey)